Src/cql-lm/schema/elm/clinicalexpression.xsd

```diff
@@ -547,7 +558,8 @@ The second argument is expected to be of type CodeSystem. When this argument is
 			<xs:extension base="OperatorExpression">
 				<xs:sequence>
 					<xs:element name="codes" type="Expression"/>
-					<xs:element name="codesystem" type="Expression"/>
+					<xs:element name="codesystem" type="CodeSystemRef" minOccurs="0"/>
+					<xs:element name="codesystemEx" type="Expression" minOccurs="0"/>
```

Comment:
Are you suggesting the name of the new element be `codesystemExpression` rather than `codesystemEx`?

Reply:
Yes. I'm imagining scenarios where someone says valueSet AND valueSetExtension / Extended / some other Ex word. Since FHIR allows extensible ValueSet bindings I'm thinking there's the possibility for some conceptual confusion here.

Reply:
Agreed and applied